tests/test_orchestrator_streaming_output.py

```diff
@@ -14,10 +14,11 @@
 
 FileService(file_manager_config)
 
+
 def strip_lines(text):
     return "\n".join([line.strip() for line in text.strip().split("\n")]).strip()
 
-
+@unittest.skip("Entire test class skipped due to hanging tests")
```

Comment:
This should be removed now that I have merged in the fix for this

Reply:
It has been removed in #96